app/src/components/pages/collectionLanePage/collectionLanePage.tsx

```diff
@@ -0,0 +1,65 @@
+"use client";
+import {
+  Box,
+  Heading,
+  HorizontalRule,
+  SimpleGrid,
+} from "@nypl/design-system-react-components";
+import PageLayout from "../../pageLayout/pageLayout";
+import { useNumColumns } from "../../../hooks/useNumColumns";
+import { useParams } from "next/navigation";
+import { headerBreakpoints } from "../../../utils/breakpoints";
+import { slugToString } from "../../../utils/utils";
+import CollectionCard from "../../../components/cards/collectionCard";
+import { CollectionCardModel } from "../../../models/collectionCard";
+import useBreakpoints from "../../../hooks/useBreakpoints";
+import CollectionDataType from "../../../types/CollectionDataType";
+import { mockCollections } from "../../../../../__tests__/__mocks__/data/mockCollections";
+import React from "react";
+
+export default function CollectionLanePage() {
+  const params = useParams();
+  const { isLargerThanLargeTablet } = useBreakpoints();
+  const slug = params.slug as string;
+  const title = slugToString(slug);
+  const numColumns = useNumColumns();
+  return (
+    <>
+      <Box
+        sx={{
+          display: "flex",
+          flexDirection: "column",
+          "> h1": {
+            marginBottom: 0,
+          },
```

Comment:
Why not add this style directly as a style prop to the `Heading` component?

Reply:
Sorry holdover from `Heading`s displaying as an `hgroup`. Will fix at next chance